.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
- [ ] It would be better if we implement the functionality where the background changes based on the received data